DEVELOPING.md

```diff
@@ -101,6 +101,13 @@ You will still have to fix a few things:
 * Uncheck the "Allow output directories per source directory" in Build path
 * Add transitive project dependencies in Build path
 
+## Preparing a Pull Request
+
+One common build failure is code styling.  Reproduce results locally with:
```

Comment:
SJS doesn't do double spaces. Please be consistent with the rest of the file.

Reply:
👍